openshift/scripts/generate-dockerfiles.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# This script generates the productized Dockerfiles
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function install_generate_hack_tool() {
+  tmp_dir=$(mktemp -d)
```

Comment:
The var could be local and the rhs expression should be in `""` (in case some psychopath has spaces in dire names).

Reply:
I mean mktemp probably won't generate spaces in path, but still.

Reply:
Hmm I just tested and even without "" splicing is not happening, weird.

Reply:
good point. Updated it